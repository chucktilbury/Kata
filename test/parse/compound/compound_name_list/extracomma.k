;-------------------------------------------------------------------------------
; Demonstrate that a extra ',' causes an error
;-------------------------------------------------------------------------------

func name ()() {

    func_ref()(this.is.a.name,, whoopd, asd.qwe.rt)
}
