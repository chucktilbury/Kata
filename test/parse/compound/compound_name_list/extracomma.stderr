Syntax: /home/chuck/Src/Kata/test/parse/compound/compound_name_list/extracomma.k: 7: 32: expected a compound name but got ,
Syntax: /home/chuck/Src/Kata/test/parse/compound/compound_name_list/extracomma.k: 7: 32: expected an optional list of names but got ,
Syntax: /home/chuck/Src/Kata/test/parse/compound/compound_name_list/extracomma.k: 7: 32: expected function body element but got symbol
Syntax: /home/chuck/Src/Kata/test/parse/compound/compound_name_list/extracomma.k: 7: 32: expected function body but got symbol
Syntax: /home/chuck/Src/Kata/test/parse/compound/compound_name_list/extracomma.k: 7: 32: expected module item or end of file but got func
Syntax: /home/chuck/Src/Kata/test/parse/compound/compound_name_list/extracomma.k: 7: 32: module cannot be empty
